mathesar/database/base.py

```diff
@@ -1,13 +1,15 @@
 from django.conf import settings
 
 from db import engine
+import logging
 
+logger = logging.getLogger(__name__)
```

Comment:
It doesn't look like we're using the logger anywhere, can you remove it for now?

Reply:
haha...done removed i. Used it for local logging.